src/test/resources/pages/adminEmailCompose.html

```diff
@@ -347,7 +344,7 @@
                                 </span>
                             </form>
     
-                            <div id="documentBaseUrl">${app.url}</div>
+                            {*}
```

Comment:
Here's what's happening: take a look at `compose.tag`:
![image](https://cloud.githubusercontent.com/assets/7261051/9561747/2f04d046-4e85-11e5-95eb-9f60557acaa4.png)
and `AdminEmailComposePageData`:
![image](https://cloud.githubusercontent.com/assets/7261051/9561753/50b75e84-4e85-11e5-9288-043978530a2d.png)
The above will be non-nullified only if there's an email to edit. Otherwise, you're trying to get a field from a null object.
So, final verdict: you can't wildcard this. Should modify the code such that the `appUrl` will be returned regardless of existence of email to edit.